qplist.pro

```diff
@@ -0,0 +1,14 @@
+QT += xml
```

Comment:
I mean, it's not like I am trying to impose rules over your head, this is a community thing. If you are ultimately convinced that "qt" is the right thing (despite the things I mentioned above) I will strongly consider this.

I would even go as far and remove any prefix (e.g. plist), or add your own prefix (rwplist). Dunno.